wcc-contentful/lib/wcc/contentful/active_record_shim.rb

```diff
@@ -7,6 +7,26 @@ def attributes
     @attributes ||= to_h['fields'].tap { |fields| fields['id'] = id }
   end
 
+  def cache_key
+    key = "#{self.class.model_name}/#{id}"
+    key += "-#{cache_version}" unless ActiveRecord::Base.try(:cache_versioning) == true
+    key
+  end
+
+  def cache_key_with_version
```

Comment:
why 2 methods?

Reply:
I was following this blog: https://blog.bigbinary.com/2019/08/06/rails-adds-support-for-recyclable-cache-keys.html

Reply:
https://api.rubyonrails.org/classes/ActiveRecord/Integration.html#method-i-cache_key_with_version